nnvm/python/nnvm/frontend/onnx.py

```diff
@@ -824,6 +898,16 @@ def from_onnx(self, graph, opset):
             else:
                 self._num_input += 1
                 self._nodes[i_name] = _sym.Variable(name=i_name)
+        # get list of unsupported ops
+        unsupported_ops = []
+        for node in graph.node:
+            op_name = node.op_type
+            if op_name not in supported_ops:
```

Comment:
checking op_name in identity_list or op_name in convert_map or op_name = 'Constant' should be enough